moveit_ros/moveit_servo/test/config/servo_settings.yaml

```diff
@@ -27,6 +26,12 @@ publish_joint_positions: true
 publish_joint_velocities: false
 publish_joint_accelerations: false
 
+## Smoothing plugin
+# The default plugin is a Butterworth lowpass filter that requires this parameter.
+# It must be >1.0, typically in the range 1.0 < coefficient < 100.0.
+# If you use a different plugin, replace with the relevant parameters.
+low_pass_filter_coeff: 2.0  # Larger --> trust the filtered data more, trust the measurements less.
```

Comment:
It doesn't look like servo_params allows for `smoothing_filter_plugin_name` to not be set? I would replace the low_pass_filter_coeff with the plugin name since we aren't parsing low_pass_filter_coeff right now.

Reply:
The items in this "test" folder just hadn't been updated yet. Done.